test/code/plugins/out_oms_api_systest.rb

```diff
@@ -0,0 +1,62 @@
+require 'fluent/test'
+require_relative ENV['BASE_DIR'] + '/source/ext/fluentd/test/helper'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/oms_configuration'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/out_oms_api'
+require_relative 'omstestlib'
+require_relative 'out_oms_systest'
+
+class OutOMSApiTest < OutOMSTest 
```

Comment:
Good catch. I thought the test_send_data method in out_oms_api_systest will override the test_send_data in out_oms_systest, but it's not true. I'll refact the common code for out_oms system tests into a base class, and make all out_oms_xxx system tests derive from them.